examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_1.ipynb

```diff
@@ -0,0 +1,233 @@
+    "\n",
+    "```python\n",
+    "import pandas as pd\n",
+    "```\n",
+    "\n",
+    "to:\n",
+    "\n",
+    "```python\n",
+    "import modin.pandas as pd\n",
+    "```\n",
+    "\n",
+    "Changing this line of code will allow you to use all of the cores in your machine to do computation on your data. One of the major performance bottlenecks of pandas is that it only uses a single core for any given computation. Modin exposes an API that is identical to pandas, allowing you to continue interacting with your data as you would with pandas. There are no additional commands required to use Modin locally. Partitioning, scheduling, data transfer, and other related concerns are all handled by Modin under the hood."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 1: How to use Modin\n",
+    "\n",
+    "**GOAL**: Learn how to import Modin to accelerate and scale pandas workflows."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p style=\"text-align:left;\">\n",
+    "        <h1>pandas on a multicore laptop\n",
+    "    <span style=\"float:right;\">\n",
+    "        Modin on a multicore laptop\n",
+    "    </span>\n",
+    "\n",
+    "<div>\n",
+    "<img align=\"left\" src=\"../../../img/pandas_multicore.png\"><img src=\"../../../img/modin_multicore.png\">\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Concept for exercise: Dataframe constructor\n",
+    "\n",
+    "Often when playing around in pandas, it is useful to create a DataFrame with the constructor. That is where we will start.\n",
+    "\n",
+    "```python\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "frame_data = np.random.randint(0, 100, size=(2**10, 2**5))\n",
+    "df = pd.DataFrame(frame_data)\n",
+    "```\n",
+    "\n",
+    "When creating a dataframe from a non-distributed object, it will take extra time to partition the data for Modin. When this is happening, you will see this message:\n",
+    "\n",
+    "```\n",
+    "UserWarning: Distributing <class 'numpy.ndarray'> object. This may take some time.\n",
+    "```\n",
+    "\n",
+    "Modin uses Ray as an execution engine by default. Since this notebook is related to OmniSci, let's run examples on the OmniSci engine. For reaching this, we need to activate OmniSci either via Modin config or Modin environment variable. See more in [OmniSci usage](https://github.com/modin-project/modin/blob/master/docs/development/using_omnisci.rst) section.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Note: Do not change this code!\n",
+    "import numpy as np\n",
+    "import pandas\n",
+    "import sys\n",
+    "import modin\n",
+    "import modin.config as cfg\n",
+    "cfg.StorageFormat.put('omnisci')"
```

Comment:
let's place configuration in a separate cell for more relation with the text above

Reply:
> let's place configuration in a separate cell for more relation with the text above.

makes sense

> And probably add suggestions about other ways of configuration like here [[1]](https://github.com/Rubtsowa/modin/blob/ba4d6a25405e34900538abd6cf461f43de418f47/examples/tutorial/jupyter/execution/pandas_on_dask/local/exercise_1.ipynb?short_path=328db54#L69-L81)

We have a link to OmniSci usage section [here](https://github.com/modin-project/modin/pull/4001/files#diff-28ec1eb8554320939e91143a5c351a2d69d1c283daa67d5306f570699b75e2d3R81). I think this should be enough.

Reply:
@YarShev @dchigarev I moved this lines in separately section